bibcheck/plugins/ref_jvolume.py

```diff
@@ -0,0 +1,37 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2013 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+"""
+For journal reference field 999C5s
+move letter in front of volume if volume > vol_min or volume < vol_max
+"""
+def check_record(record, journal, letter, vol_min=0, vol_max=999999):
+    """
+    Move volume letter in front of volume
+    """
+    import re
+    jvol = re.compile("^(%s,)(\d+) *%s(,.*)$" % (journal, letter))
```

Comment:
Also you may make this regexp and its usage more clear if you use named element:
```python
jvol = re.compile("^(?P<journal>%s,)(?P<volume>\d+) *%s(?P<page>,.*)$" % (re.escape(journal), re.escape(letter)))
```
The you can simply refer to them via `result.group('journal')` etc.

Actually I am not sure how to name the above regexp component because I am not sure what they represent, so even better to employ the named technique :+1: